### ISSUE TITLE
Wah Ki is too much of a warzone, in fiction and for capping

### ISSUE THREAD

**Issue by Terin** [endless-sky/endless-sky#5893]
**Is your feature request related to a problem? Please describe.**
Currently, Wah Ki is one of the biggest bloodbaths of a system in all of Endless Sky, where it only feels eclipsed by something like Mesuket. While the latter feels appropriate, due to the Kor Sestor and Mereti being automatons who have been let loose to consume their regions of space for resources, the Unfettered are runaways and dissidents, making a living on three systems. The amount of personnel and resources required to sustain this war effort is _absurdly_ out of proportion for how they're set up in fiction, especially around the time of <first> <last>, where they have an actually feasible target of their old worlds in Wanderer space.

On top of it making little sense in-world, Wah Ki is also an extremely abusable system when it comes to capping, along with Mesuket. However, while the Automata are limited by not being part of the capping ladder and having the self-destruct chance, making farming them en masse a pain, Shield Beetles in Wah Ki are trivial to obtain once you have a Star Queen, Heavy Warship, or even Medium Warship if you're lucky. At that point, you've got a ship more powerful than anything in human space, with up to 215 bunks, and easy access to as many as you want.

**Related Issue Links**
None specifically related to this issue, only general talks about Wah Ki mechanically.

**Describe the solution you'd like**
The fleet frequencies of Large Hai/Large Unfettered should be drastically reduced, or the period increased, to reduce the amount of those ships in the system. Having a system mostly populated by Hai/Human merchants, which may be harassed by a fleet of Unfettered occasionally, until they can be overwhelmed by those merchants or pushed out by a main wave of Hai warships, would feel much more appropriate to what the Unfettered are. As it stands, their current behavior is not sustainable for how they're set up.

**Describe alternatives you've considered**
A nerf of the Shield Beetle in some ways could help reduce them being a clear winner for capping. As they stand, they're more powerful than Korath ships, with tons of cargo space, outfit space, and bunks to boot. They could have their cargo/spare bunks slashed in half, or more, to help with capping, or reduce their combat stats to reduce the power they contribute there. However, this would not address the fictional issues.

If we had to keep the fleets, the introduction to Hai space, along with various missions should either heavily hint to, or outfight state that the Unfettered have begun an incredible war effort not seen in hundreds of years. Hai with a vested interest in keeping the peace could downplay it, but something should reflect the amount of carnage going on in Wah Ki. However, this would not address the easy train of disable Shield Beetles the player could capture.

**Comment by Zitchas** [comment]
Here's another alternative: Implement a personality similar to "Coward" except that also applies to the the flagship, thus, *all* the ships in the fleet will flee the battle when their shields hit zero. If all the Unfettered and Hai run away as soon as their shields drop, then the amount of 'easy capture" ships in Wah Ki should drop to much more reasonable levels. I'm sure there will still be some, but that's fine. 

This *also* solves the whole resource issue, since constantly repairing all their warships is a much more understandable level of consumption than having to *replace* that many warships.

**Comment by Vedvod** [comment]
An alternative could be just to make them sometimes self-destruct like marauders or automata. It could be a "don't let the enemy benefit from my loss" kind of thing, and also a "we'll take you down with us" or something.
Another possibility I can see could be a modified version of a shield beetle, used by the unfettereds (unfettered shield beetle?) that, as suggested, has less cargo and outfit space, or is lacking in some other way that doesn't detract from their combat, but does detract from other applications.
Another idea I had (which is the worst one) could be to have a modified version of the shield beetle that has high shields, low hull, and sometimes explodes when it runs out of shields, "the Unfettereds didn't have the resources, so they made a lot of cheap but highly volatile shield generators" or something.
As Zitchas suggested, they could just flee combat when low shields. That's probably a much better solution than any of mine.

**Comment by Terin** [comment]
@Zitchas That's definitely a general wishlist for merchant fleets, as well. When I did a pirate playthrough, I was _flabbergasted_ at how much merchants wanted to duke it out with my fleet of 20 pirate cruisers. Although I guess they'd more just avoid fights in general, rather than get into them and retreat when a weak link gets low, which would be more handy for the pirate/Unfettered fleets, yeah.

@Vedvod I think an Outfit Variants PR is finally in the works, so seeing "Cheap" versions of normal Hai outfits there would be great. EV Nova had those as well, and they were really flavor, and being able to see them in practice in Unfettered spaceports/ships would be really cool.

But as to both these points, those mostly address the mechanical impact with new code additions. Fictionally, making it so the ships would retreat would be _better_, but it still means the Unfettered are jumping in with _tons_ of Shield Beetles into an active warzone when they should not have that capability in the first place. As is, they'd likely not be able to get away with that much, seeing as how much overkill trackers can do.

While both of those are solutions I'd like in the future, they'd require new personalities/outfit attributes in code, while my suggestion is just increase the period of Large Hai/Large Unfettered by like, 5-10. _Much_ easier.

**Comment by Amazinite** [comment]
I will note that I opened #4039 in regard to fleet frequencies in human space being too high. but we could certainly expand those change to the rest of the galaxy as described in #5779, or at the very least take an assessment of fleet frequencies in Hai space as well.

Something I will make note of is how the Unfettered, outside of their own systems and the Wanderer systems during the campaign, only spawn in Wah Ki. Any further than that and the only reason they're there is because you let them spawn in Wah Ki and jump further into Hai space. So what we could perhaps do is generally reduce the frequency of ships in Wah Ki, but at the same time expand the Unfettered fleets to perhaps travel one system deeper than they currently do. Rationale behind this might be that since we're reducing the frequencies in Wah Ki as to not be an endless battle, Unfettered that do enter Wah Ki are more likely to be able to jump into neighboring systems.

**Comment by Terin** [comment]
Yeah, the Wanderer invasion doesn't feel like very much when the Unfettered regularly kill themselves in spectacular fashion out in Wah Ki. Spreading them out a jump while reducing those frequencies in general sounds good, although I don't think that's contingent on a region-wide rebalance of fleet frequencies. The raids in those systems scare commerce to some extent, so they're less trafficked than like, Greenwater or Hai-home.

Will try out some numbers and see how they feel pretty soon, and might PR it. I think changes here can be done just fine without waiting for the rest of the galaxy to be done over first.

**Comment by binaryspica** [comment]
Well, isn't Wah Ki supposed to be one of the most intense warzone in the galaxy? Also its Hai fleet always try to disrupt the boarding attempts. Although it is one of the best Shield Beetle farming area, but its quite risky as well, for Hai fleet's attack against disabled Unfettered ships and Hai Tracker of Unfettered.

I think that the current situation just represents what it should be; Hai and Unfettered Hai are keep stalemate on the system all the times, for it is the only way to enter the other Hai territory without a jump drive. Also, unlike Wanderers, although Hai are peaceful race they do know how to fight, so it is unlikely that some infiltrators with the jump drive can threatening Zuba Zub or Bore Fah either - you know, there are also Hai fleet patrols to watch over the course.

**Comment by InfiniteDonuts** [comment]
I agree 100% that Wah Ki is WAY to out-of-balance for capping - I can get myself a massive fleet with tons of Shield Beetles after spending only about an hour there. Yes, it's risky, but either it's not risky _enough_, or there are just too many huge battles there.

**Comment by binaryspica** [comment]
I do think that it is the best 'early' game farming zone. But you need to remember that the presence of Hai is already a secret. Yes you can found them without a jump drive, and I did, but for me I did it not before travel almost all explored human area. Also the possible explorers needs to endure many systems without much republic patrols to protect them from the pirates as well. Although they may had a jump drive and ease the process, but still it is not that easy anyways.


If fix the AI is the way to go, then it can be changed by add 'combat recon' fleet of Unfettered Hai, and they have coward behavior as Zitchas suggested. Those fleets are just want to keep in check the defenders of Wah Ki and ready to run, rather than the main fleet. The normal fleet with attack duty still stands, and they will fight to the death and may aim for jump to the other systems, but with this the chance to encounter the Unfettered fleet with fight to death is reduced. Mix it up the change and the fight is still intense as it does while you can't expect a free riding easily. Still it happens, sure, but you need to keep in mind the Hai fleet try to destroy your prey as well as Hai Tracker of Unfettered, as I said above.

**Comment by binaryspica** [comment]
Else just put more marines on the Shield Beetle is an another simple solution; you may board them, but you may expect a heavy resistance. It have default bunk of 95, and you may add some Bunk Room as well. Not all Shield Beetles of Unfettered are required to be come with that much marines, for it should be still allowed to the early game hijackers(for simply remove the chance to do is too cheap), but have some chance to encounter the beetles with extra Bunk Room seems not so bad I think. Fill it with the pulse rifles match with its crewmembers is required to do, as well.

After all, too abuse it may a problem, but exploit some is not so bad. The game is all about explore, trade and fight. Then why we need to remove every single advantageous points? Especially for it is only good for the player who is already learned the game well already? Is the vast majority of players are go for Wah Ki early game and board the shield beetle? If the new pilot did it, then the pilot is supposed to be the veteran gamer that choose to make a new pilot, or the pilot already read some guides. Remember that the game is all about hide the information.

**Comment by Vedvod** [comment]
> Else just put more marines on the Shield Beetle is an another simple solution; you may board them, but you may expect a heavy resistance. It have default bunk of 95, and you may add some Bunk Room as well. Not all Shield Beetles of Unfettered are required to be come with that much marines, for it should be still allowed to the early game hijackers (for simply remove the chance to do is too cheap), but have some chance to encounter the beetles with extra Bunk Room seems not so bad I think.

One of Terin's issues was with the fact that the Unfettereds could not possibly sustain a large population to pilot the beetles and such. Adding more crew to the Shield Beetles, while making boarding more difficult, would still not discourage a city-ship boarding strategy or fit with the lore problem that Terin outlined.

Also, I don't really feel that "only veterans would know about it" would really work as a justification for not changing it as an exploit. There is the planned Hai-space alternative start, which would make this really early game, and there are probably other reasons that it would be negative to keep it.

**Comment by binaryspica** [comment]
Isn't it possible that Unfettered are reduce the mouth to feed by send their people to Wah Ki? I am sure that no one will be surprised even if they did, although I wonder they are actually intended to do.

After all their fierce attitude in combat means it is nothing wrong to fight to the death. And you know the quote, the disabled Unfettered ship sometimes says 'come here and board me' means they are also enjoy hand to hand combat as well, so it is not so weird that some of their ships are quite well prepared for boarding.

Anyway, if the presence of Hai can be more easily revealed, it is nothing wrong to reduce the chance to capture the shield beetle. Although simply block the chance at all is not a good move.

**Comment by MasterOfGrey** [comment]
I will add here that one suggestion for helping to handle this that has been discussed and that someone started making a ship for, was to add a secondary layer to the combat. 

This would involve having a number of drone carriers with drones that are a neutral third government that just fly around collecting debris (and "survivors in escape pods") and which will randomly either land on Cloudfire or run away to unfettered space depending on which side's electronic warfare systems have wrested control of the drone carrier at the time. This would help to account for the loss of life and resources by actively mitigating it.

This would work particularly well in tandem with Zitchas' alternate personality suggestion, and not require any reduction in fleet spawning rates.

**Comment by Terin** [comment]
While it's somewhat of a problem how many lives they throw away, that's not nearly as much of an issue as how many perfectly functional heavy warships they throw away in a fruitless conflict. It more than stretches disbelief that, with three systems, they can regularly throw out these powerful ships with no gain.

The Hai being obfuscated from a starting player definitely makes it not universally game-breaking. However, they are almost certainly the first main faction discovered by the player, and they're notified how the Unfettered are up north. Most players will eventually check that out, and can pretty quickly see the wealth of ships left behind within no time. 

By no means do I want to just remove any cool discoveries for experienced players to take advantage of. I love that Jump Drives are a thing that exist in universe, and you can upgrade to them if you know where to find them. I just want to tone down the very easy source of dozens of heavy warships. In fact, lowering the military frequency could mean that you, the player, could take the fight into your own hands and try to disable Shield Beetles yourself, rather than having them mostly overkilled by the massive Hai fleets and picking up the (plentiful) scraps.

Raising the default bunk count on Shield Beetles might be something, but 95 bunks is still pretty capable by the 7 million credit Star Queen. Maybe then that means we go down the line and nerf the Star Queen, or rather than cascading changes in one direction, we just slow it down at the source.

**Comment by Terin** [comment]
@MasterOfGrey Unless the Unfettered are able to get away with like, 90%+ of their ships intact, I still find it extremely unbelievable that they can manufacture such a vast amount of warships only to send them to die in a pointless battle. To pull from Escape Velocity Nova, they're like the Auroran Empire on steroids, except that they do not control 60% of all known space, they control _three systems!_

We can address this with code changes (Zitchas's suggested personality) on code changes (escape pods) on code changes (rescuing pods/salvaging debris) on codes changes (appropriately fleeing to their home system)... or we do the simple change, reduce the number of ships in the area, and handle this balancing pain-point fictionally and mechanically.

**Comment by MasterOfGrey** [comment]
Yes I am aware, and I am in favour of `code changes (Zitchas's suggested personality)`, content/fluff changes: (rescuing pods/salvaging debris) and `codes changes (appropriately fleeing to their home system)`

**Comment by Terin** [comment]
I'm definitely for those code changes in the broad sense, to be clear. They'd be a great addition to game in a bunch of different ways. But to have a problem only solved by multiple layers of code solutions, that we don't know if they'd get approved, how long they'd take, or if anyone would even develop, when a _much_ cleaner solution exists feels extremely silly to me.

**Comment by gunqqer** [comment]
It seems the main issue is with how ES has battles that don't belong in lore. Pirates endlessly throwing ships at you when you have cargo, durring free worlds the Republic and FW just sending ships and keep sending ships even when there isn't a major battle taking place in lore. There is just a lot more fighting than you think you'd see in space. The only place that makes sense is Meskut where it really is two factions endlessly throwing ships at each other. This instance is just one that can be easily abused to get heavy warships within a year or starting. Which means thse changes could be generalized to many other systems to tone down the amount of combat governments do.

**Comment by MasterOfGrey** [comment]
> But to have a problem only solved by multiple layers of code solutions, that we don't know if they'd get approved, how long they'd take, or if anyone would even develop, when a much cleaner solution exists feels extremely silly to me.

Most coding changes here seem to come about directly as a result of demand. Opting for the "simple" solution detracts from the demand, so I would be in favour of not half-assing it.

> ES has battles that don't belong in lore

Yeah, having the personality for retreat would be really good, then most of those would become skirmishes where people bug out again rather than full battles.

**Comment by arkhne** [comment]
I would like to add my thoughts to this.

Firstly, I think Zitchas' idea to make them flee the system as soon as their shields drop would be good. As would reducing fleet frequency on both sides.

Secondly, the Shield Beetle could *definitely* use a nerf in its non-combat stats. Since capping them is a problem, this leaves Cargo Capacity. I'd recommend dropping that by at least 20, possibly even 40. That is the equivalent of 26 or 53 Outfit space required to retain the same level of Cargo Capacity, so it's not even a huge loss, but I believe it helps bring them in-line with other Heavy Warships.

Thirdly, the Unfettered (along with Pirates) need their own personal answer to H2H. In the case of human Pirates, I'd recommend making them a special 'Blunderbuss' H2H outfit that is basically an even more defense-biased version of the Laser Rifle, with some lore fluff about paying homage to ancient pirate epics. In the case of the Unfettered, this would be a specialized Automated Defense Outfit, akin to Security Stations or Biodefenses (but definitely better than Security Stations).

I also have a Branch of ES for severely crippling minmaxing for capturing, but it needs outfit images before I can release it for testing. I was also hoping to add the Blunderbuss and Unfettered Defense Outfit to it before that, but hadn't gotten that far. The short of it is, all ships come with an 'Armory Capacity' attribute (I've even enabled displaying this to the player), and all H2H weapons require an amount of 'Armory Capacity' (IE: Rifles = -1, Nerve Gas = -4), along with an outfit that increases the 'Armory Capacity' outfit. This means a general loadout of 'just add bunks and rifles' is doable, but if you want to minmax, you will have to make concessions. Do you take more Bunks and just use Rifles? Or use less Bunks to take more Grenades/Gas? Or try to find a balance of both.

The upside of all these changes/suggestions is that it is all purely Content, with the minor exception of displaying the Capacity, which as I said, has already been done. So, there's definitely some code-free options here. Personally, I would do all of these at once, but perhaps I am biased against the Shield Beetle, as it makes the Albatross look stupid as a Heavy Warship from a people based on looting ships, that doesn't even perform that job better than a standard Warship from the Hai.

**Comment by Terin** [comment]
> Most coding changes here seem to come about directly as a result of demand. Opting for the "simple" solution detracts from the demand, so I would be in favour of not half-assing it.

That _really_ feels like "Perfect is the enemy of good". We don't have to have easily-solvable gaping wounds in the game just as a carrot on the stick to implement a bunch of complicated fixes, especially when those fixes have plenty of other applications. Like I mentioned above with my pirate playthrough, human Merchant fleets get extremely bogged down in fights since their fleets are in system, even the "coward" ones due to the AI's susceptibility to pile onto damaged ships. That personality fix would be great for human pirates, which take up a large part of space and the player's experience, to the point where those fixes will almost certainly come when we had someone capable/motivated enough to implement those changes.

And again, the amount of Shield Beetles the Unfettered send into Wah Ki is already stretching to the point of being unbelievable. If they take _any_ losses of note (which will happen with the Hai sending barrages of Trackers to obliterate ships), it just becomes absurd that the Unfettered aren't burning their materiel down to nothing.

> Secondly, the Shield Beetle could definitely use a nerf in its non-combat stats. Since capping them is a problem, this leaves Cargo Capacity. I'd recommend dropping that by at least 20, possibly even 40. That is the equivalent of 26 or 53 Outfit space required to retain the same level of Cargo Capacity, so it's not even a huge loss, but I believe it helps bring them in-line with other Heavy Warships.

Yeah, this feels like a really easy knob to turn down. A Shield Beetle pretty much has no weakness, it doesn't also need to be an extremely capable freighter! Especially when the Hai are already pretty well covered with cargo haulers.

> I also have a Branch of ES for severely crippling minmaxing for capturing, but it needs outfit images before I can release it for testing. I was also hoping to add the Blunderbuss and Unfettered Defense Outfit to it before that, but hadn't gotten that far. The short of it is, all ships come with an 'Armory Capacity' attribute (I've even enabled displaying this to the player), and all H2H weapons require an amount of 'Armory Capacity' (IE: Rifles = -1, Nerve Gas = -4), along with an outfit that increases the 'Armory Capacity' outfit. This means a general loadout of 'just add bunks and rifles' is doable, but if you want to minmax, you will have to make concessions. Do you take more Bunks and just use Rifles? Or use less Bunks to take more Grenades/Gas? Or try to find a balance of both.

That could help in some ways, but it's kind of the "issue" with Arfectas: They've got incredible defense, but not that many crew on them. Every Pug you kill in combat drops their defenses a drastic amount, making follow-up kills even easier. 47 crew isn't a large amount. It'd be even easier once you get your first Shield Beetle as well, since it can get over 200 bunks at the moment. It's definitely worth considering, especially to make H2H not just "get 1 nerve gas/frag grenade/pug stick for each crew member", but I feel like we should attack it from both angles, by toning down the quantity of ships available as well.

**Comment by binaryspica** [comment]
I have killed countless pirates and Korath Exiles and I am sure that I have killed about a six figure numbers of them(entity, not ship) total, but the enemy are still comes from nowhere. Strictly count the losses is pointless. The aggressiveness of Unfettered Hai is the only point to consider. That's why I said that fill it with full of marines is a solution; in the gameplay, some loss of the life won't affects the world at all. And isn't Unfettered the most aggressive faction in the world?

Also, since Hai is not so aggressive race, it is natural that Shield Beetle comes with some amount of spare cargo holds. If you want to use it aggressively, you may take some outfit expansions. 

And, seriously, you need to consider that various shield beetles already available to the users. You can't manually edit all the stats of ships already existed through the players' save files for the data of the each ships are unique and each of their stats are recorded in the save files individually. As my Gascraft is still a 'Gascraft', not Puffin. Nerf or buff the existed ships is something we can't handle well.

I think that an another idea is to increase the capture defense of pulse rifles. You know, players are rarely needs for capture defense, and we are usually against the opponents' capture defense. I wonder that it helps much since Human already has fragmentation grenade, but nonetheless it increases the casualty or required marine points to capture the ship. It won't change the tide, but it may cause a small change.

**Comment by Terin** [comment]
> I have killed countless pirates and Korath Exiles and I am sure that I have killed about a six figure numbers of them(entity, not ship) total, but the enemy are still comes from nowhere. Strictly count the losses is pointless. The aggressiveness of Unfettered Hai is the only point to consider. That's why I said that fill it with full of marines is a solution; in the gameplay, some loss of the life won't affects the world at all. And isn't Unfettered the most aggressive faction in the world?

Yes, there are theoretically infinite humans, Quarg, Hai, Korath, Remnant, Coalition, Wanderers, and so on and so forth due to the way the game is built That's most video games, you can't kill everyone in the world in a GTA game. However, the snapshots we get of traveling around the galaxy are how we get any meaning from that. You look at Mesuket, you've got infinite waves of gigantic, deadly automata fighting over an absolute nothing of a system, in one of the most glorious spectacles of destruction in the game, which fits how out of control they've gotten. You've got the Coalition, with very bustling merchant activity in their peacetime, but a proportional amount of ultra-powerful warships to keep an eye on them. In the Ember Wastes, you can have Korath and Remnant passing by ships in the night, sometimes getting into skirmishes before jumping or cloaking away, or the Ka'het, looking completely identical but aggressively tearing at each others with no discernible rhyme or reason, to show what's become of the Builder's tools.

You get to the Unfettered... and you get too much at once. Obviously they're made up. Obviously they can't _actually_ mine out their systems, unless someone writes a mission that says they're running low. But the picture we are given in game is so out of contrast with everything else, where it sounds like to the Hai they're just a wayward nuisance who mostly make Cloudfire a bit of a miserable place... except for how willing they are to obliterate them. The aggressiveness they have with the Wanderers? That works great, since they're suddenly feeling like this bottled up force that has finally been let loose, since they have everything to gain by getting their old worlds back. But that doesn't work on the other side.

> Also, since Hai is not so aggressive race, it is natural that Shield Beetle comes with some amount of spare cargo holds. If you want to use it aggressively, you may take some outfit expansions. It was already called versatile ship, so on the peacetime it can be used for auxiliary cargo ship as well, so I think that it is fine that it have some amount of cargo.

If you want to use it aggressively, you just use the ship as-is, since it already has so much outfit space by default. I think it might even be in the top 10, from a quick glance and depending on how pedantic you'd want to get with Kestrel variants. Then that's on top it having like, the 4th most cargo space for heavy warships, only beaten out by the Bactrian, Raider, and Skylark, which are pretty notable selections. Reducing the cargo space when it has pretty much no other weakness is absolutely warranted.

> And, seriously, you need to consider that various shield beetles already available to the users. You can't manually edit all the stats of ships already existed through the players' save files for the data of the each ships are unique and each of their stats are recorded in the save files individually. As my Gascraft is still a 'Gascraft', not Puffin. Nerf or buff the existed ships is something we can't handle well. Only thing we can do something about the ship is to make a new ship. If you really want to do something with Shield Beetles' stats, the only solution is to make an 'Unfettered Shield Beetle' class which is something new. Well, we have seen Marauder ships already, so it is not a new concept, though.

The Gascraft -> Puffin transition was merely for the name, stats aren't nearly as much of an issue. Yes, if your pilot has some ships when a new update nerfs that ships, they will be more powerful than the new ones. However, that hasn't stopped us before from tweaking ships like the Mule. I feel like the Shield Beetle is in a similar jack-of-all-trades place as that, and I'm not even asking to reduce the handling or shields. Just take some cargo off the top.

> I think that an another idea is to increase the capture defense of pulse rifles. You know, players are rarely needs for capture defense, and we are usually against the opponents' capture defense. I wonder that it helps much since Human already has fragmentation grenade, but nonetheless it increases the casualty or required marine points to capture the ship. It won't change the tide, but it may cause a small change.

arkhne's armory stats for H2H gear mentioned above is along the similar lines, yeah, and I definitely want to see that happen in game. However, there's only so much it can do to prevent someone from capping one Shield Beetle. I threw my poor Emerald Sword again the Worldships before they were replaced with Dredgers, I'd know! But once you cap that one, you're able to turn that ship around on ships of the same class, and really start to snowball out of control. It's worth implementing, but it's only one vector to handle problems with capping.

**Comment by binaryspica** [comment]
It was already explained well that Hai did defend the onslaught fairly well while Wanderers are struggle to do -  Hai are a peaceful race but they know how to fight and are will fight if they have to, and they are able to fend off Alpha as well. Hai are described as peaceful but they will fight off any threat. But Wanderers are not used to fight and they only developed the technique to fight recently, and even thinks that give up their lands to the Unfettered seems not so bad either for they think that their purpose is to restore the barren worlds and let the other races to enjoy it. You will remember that despite of very advanced technology Wanderers they are very slow to develop the weapons(usually dig the old designs), although those weapons are very deadly compared by the weapons of Hai due to their higher tier.

I don't think that reducing the cargo would make any positive results because of the reason as I said above; it is the ship of the peaceful faction so having more cargo is well suit for the fluff. Also it reduces the possible number of outfit expansions, reducing the overall performance. it seems that the unfettered just use it because the pattern was developed before the schism.

And why not to make an Unfettered Shield Beetle if it is problematic that much? Then unless you are enemy of Hai or buy the ships from them, all you can board is the Unfettered version, not default shield beetle. Boarding the shield beetle to get it too easy is the problem on the thread, then make the other type of shield beetle that is different with normal one would be the better idea I think. With this you may make it either far difficult to board, or can board but without much gains, while don't need to touch the shield beetle that is already available to the Hai shipyards for credits. It is far easier than just change the shield beetle itself, and we are already have such exception by Marauder variants. Although Marauder is simply the better one, but you may make the weaker or cheaper one. Hai already made a totally new ship, Solifuge, and I don't think that they care for cargo much. Else, they may consider some cargo holds because they are the pirates, perhaps.

Although it obviously looks quite cheesy, it is also possible to spamming Security Stations on the shield beetles of Unfettered if you really, really want them to be hard to boarding. I don't think that it looks good, though.

An another odd idea is, give more Hai Tracker to the Hai fleets defending Wah Ki, and assign a stationary fleet to guard Cloudfire with this. Hai Tracker is very fast missile so more tracker means you will have a hard time to actually have a chance to get to the disabled enemy shield beetles.

**Comment by USMC-ADAMS** [comment]
Been playing Endless Sky a very long time now, here are my thoughts on how to balance this. 

1) As for the fiction aspect, totally agree. The system has too much combat
2) As for the idea of simply less ships fighting being more balanced, it doesn't fix the second major core problem: Shield Beetles are too easy to get an instantly remove any challenge of the game after you get them. 

Ideas to fix #2. Shield Beetles are amazing ships. They are TOO good for how easy they are to get. They survive battles very often disabled, and Unfettered can easily be bribed after you capture one or try to get near it. They need either more crew and more average crew, or more defense. You should not be able to cap a Shield Beetle with a Clipper with nerve gas, while sitting afk in the system. 

A more preferable and perhaps easier for developers option is to simply: **gatekeep the wormhole with some type of challenge (or simply jump drive)**. It's already super easy to grab a jump drive in a large warship once you realize that 4-6 heat weapons will overheat it and it can't jump. So make that the only option. Perhaps include a bonus option where large marauder ships (whom already have alien regen) will have Keystones for wormholes, giving an additional way to unlock that territory.

**Comment by binaryspica** [comment]
> Been playing Endless Sky a very long time now, here are my thoughts on how to balance this.
> 
> 1. As for the fiction aspect, totally agree. The system has too much combat
> 2. As for the idea of simply less ships fighting being more balanced, it doesn't fix the second major core problem: Shield Beetles are too easy to get and instantly remove any challenge of the game after you get them. You shouldn't be able to cap one in a Clipper with Nerve Gas.
> 
> Ideas to fix 2: Shield Beetles are amazing ships, obviously. They are TOO good for how easy they are to get. They survive battles very often disabled, and Unfettered can easily be bribed after you capture one or try to get near it. They need either more crew and more average crew, or more defense. You should not be able to cap a Shield Beetle with a Clipper with nerve gas, while sitting afk in the system.
> 
> A more preferable and perhaps easier for developers option is to simply: **gatekeep the wormhole with some type of challenge (or simply jump drive)**. It's already super easy to grab a jump drive in a large warship once you realize that 4-6 heat weapons will overheat a Korath Raider and it can't jump. So make that the primary option, with finishing the campaign the other, longer one. Perhaps include a bonus option where large marauder ships (whom already have alien regen) will have Keystones for wormholes, giving an additional way to unlock that territory.
> 
> I love Endless Sky but things like being able to cap a Shield Beetle in 30 minutes makes the 'integrity' of the universe feel a lot cheesier. A shield beetle will manhandle a Korath Raider but at least capping a Korath Raider is very difficult in comparison.

It is impossible to change that the player sits idle and wait for something is disabled, at least for current game mechanism. The only thing to bar from do it is the fact that most factions are try to destroy disabled enemy ships. And it was said that it is too exceed already.

Also, simply ban something is not so positive, nor it is creative. Making something not an optimal choice or not making it too time efficient is fine, but what about simply shut down the possibility to board? It is something never allowed to do. If they are afk for few minutes for each try and got reload several times and got a shield beetle by boarding, then so be it.

And encountering Hai is already something like a puzzle and it is almost impossible to do from the beginning by newbies who don't even aware that there is the race called Hai. The game is all about hiding informations. If you are already had enough experience, then you can do much better in almost everything.

Also, OP said that there was a plan to easier to access for Hai, such as start in the Hai territory. Blocking wormhole is totally opposite with what OP want to do.

**Comment by gunqqer** [comment]
As always, we must remember to not strive for perfection. Trying to have a perfect game will just lead to madness, as well as trying to balance for power players. There always will be people who try and game the system and that is ok. So we should balance for other reasons. Like in this case, lore wise, it doesn't make sense for endless waves of Unfettered fleets to constantly fight. So nerfing shield beetles or trying to make the Hai harder to find for new players won't really solve the issue. Having the AI act differently and fleets spawned differently will solve this issue, and will also fix the same issue that other places will have (pirate systems, etc).

**Comment by movingpictures** [comment]
Diving in with my two credits here:

I concur that it's ridiculously easy to build a fleet of shield beetles. I've been able to go from start to shield beetle in less than two months of in-game play. The shield beetles are the go-to, solve-all vessel, and any player who has discovered the Hai will presumably find creative ways to capture a Shield Beetle under current mechanics..... all one has to do is sit around and wait, with enough crew on board (Star Queen, Centipede, etc.) and capture something disabled. Instant over-balance.

I've seen various solutions proposed, like

- reducing spawn rate
- nerfing cargo
- self-destruct devices
- change of behaviour.

All of that says to me that the consensus seems to acknowledge the Shield Beetle is too damn powerful at the points of play it is available. But none of the above suggestions actually go to the simplest solution to preventing the player from obtaining a Shield Beetle too early, and that is - get this - preventing the player from obtaining a Shield Beetle too early. In other words, **eliminate the presence of the Shield Beetle in early game play.**

Rather than having both factions (Hai, Unfettered) using Shield Beetles, simply let them use a beefier version of a Lightning Bug and Grasshopper in early play. 

As it stands, the Lightning Bug is a woefully underpowered vessel. It's comparable to a Raven, which seems rather implausible as the second-best military vessel for an old species with a history of beating up the Korath. So make the Lightning Bug more potent (say at least as potent as an Ibis, maybe more), boost the Grasshopper a bit, and let the Unfettered venture into Wah Ki with those (albeit maybe at a reduced rate). Thus, the player could conceivably obtain a Lightning Bug either through capture or purchase, and use a fleet of those to work with in early to mid-game. However, a fleet of Lightning Bugs isn't likely to give a player too much power, but give them enough.

Right now, we have a mission in mid Wanderer storyline to go find some shield restoration tech, and the end result is a very boring vessel the Hai create that has some meh drones with uninspiring weapons. Why not make the end target of that quest be the shield beetle?

This way, the Shield Beetle (With minor regenerating hull tech) would come into play as the storyline is moving into perilous Kor Sestor/Kor Mereti terrain. Making the SB a purchase-only vehicle would also make it more desirable to obtain vessels like the Albatross or the Derecho when available. With the Shield Beetle, the player would have a potent fleet vessel. Heck, you could even have the Hai give one to the player for assisting them with the regen tech (and overall good service).

**Comment by Zitchas** [comment]
I was liking that up until you suggested having the hull regen tech in the Shield. Not having SB's in that battle at all is a pretty good idea, though.

I personally don't have a problem with someone managing to get an SB if they know what they are doing. I objected to people being able to easily farm an entire armada of them within the first few hours.

So, along those lines, changing the weighting of the fleets so that SB's are 1/10th as common and the Lightning Bug is much *more* common would go a long ways to making it feel better.

That being said, I still think the best solution for the **lore** is that both sides retreat when their shields drop. AKA no-one fights to the death. If no-one is fighting to the death, then a *lot* fewer disabled ships, and our problem is solved no matter what kind of ships they are flying.

**Comment by binaryspica** [comment]
Just eliminate the presence of the shield beetle is unlikely to happen, for both Hai and Unfettered are share the same design. But dramatically reduce the numbers of shield beetle of Unfettered fleet on Wah Ki on the early games can be a solution. You know, Korath Raider can be spawned on east Syndicate worlds but it rarely happens. While Shield Beetle is quite handy ship, but I don't think that it is more precious than a Korath Raider(strictly speaking, the jump drive beneath of it). 

I do think that reduce the possibility is nothing wrong, as long as it CAN be happened if you try to do anyways and the chance is not too ridiculously low. But simply prevent to do it at all is an another story.

**Comment by binaryspica** [comment]
An another solution is, **do not allow to enter Cloudfire** until you had enough reputation with Hai, just like Hai-Home. You know, Cloudfire is the frontline base of Hai fleets defending Wah Ki, and the planet was largely abandoned centuries ago. Then why not to them to say no for the foreigner, just as say no for enter their precious capital world? 

You know, the easiest way to avoid further conflict after the board is to enter Cloudfire. On there you may resupplied with fresh marines and ready to hunt an another disabled ship as well. Initiate the hyperdrive costs you to stand still for a time, make it the good prey of remaining Unfettered ships or sometimes you or your just captured ship can be struck by blind Hai Trackers. And there is the good reason to not allow to enter Cloudfire - it is the most precious frontline base to defend against Unfettered. Sure it doesn't prevent them to do it at all and it is not should be. And had enough reputation with Hai to allowed to enter Hai-Home is not so difficult task either. But at least it may slow down the process.

**Comment by USMC-ADAMS** [comment]
Just make them harder to capture. This won't mess up the game for newbies and will make playthroughs more meaningful for experienced players. 

A very simple way to do this is, as a couple other people have stated, give them extra crew defense other than pulse rifles. 

Nobody is saying that the game should be perfect. It's still always going to be possible to grab a jump drive and leave the game on for 1 hour in an Automata battlefield and basically get a free Tier 3 ship. But that will usually take at least a 'decent' timesink to achieve, or an insane amount of luck if you try to do it in a Tier 1 ship. 

You can go from a Shuttle > Clipper > Shield Beetle in 15 minutes if you get lucky. It just degrades the satisfaction of the other ways to play the game. 

Like, just give them a more balanced range of crew, instead of low or high, and give them security stations.

**Comment by movingpictures** [comment]
> I was liking that up until you suggested having the hull regen tech in the Shield. Not having SB's in that battle at all is a pretty good idea, though.

I pitched that because it would tie into the quest to get regen tech. It wouldn't have to be staggering - just enough to "fit" into that storyline. That could be offset by reducing the hull.

FYI, I tinkered with a savefile last night to see if I could bring a lightning bug up to a decent "medium warship" level in the spirit of what I was suggesting. Haven't quite got there, but I'm close-ish.

**Comment by Zitchas** [comment]
There's already a lot of comparisons with the SB and the Albatross being very similar. We don't want to start giving the SB things built into the hull too. That road leads to them becoming even more similar than they already are, and reduces the Albatrosses main advantages.

**Comment by InfiniteDonuts** [comment]
I believe the best course of action (in my opinion) would be to simply make less Unfettered fleets jump into Wah Ki. They're outgunned (three systems vs I don't know how many), and they're low on resources and personnel. Personally, if I was in that position, I'd wait a few weeks, gather resources, and launch a massive attack, instead of just kind of sending in my forces basically one group at a time.

**Comment by tehhowch** [comment]
One of the technical features in the near-term roadmap will make capturing ships in an active battlefield significantly more difficult (your ship will be vulnerable to enemy fire during the attack and plunder operations, but will not be able to defend itself).

Determined pilots will still be able to loot and capture in this system (and others like it), but it will require more strategy and not likely be feasible in the "early game," without additional cleverness.

**Comment by Zitchas** [comment]
Definitely. That again, circles around to the *there is no good lore reason for the two sides to constantly kill each other.*

Constantly probing each other's defenses? Sure, makes a lot of sense. I could even accept "the unfettered are constantly raiding the Hai to force them to maintain their military at a high state of readyness and training." But both of those involve *not being suicidal and annihilating each other.*

Sorry to sound like a broken record, but the real solution here is implementing a personality wherein the two sides retreat when their shields drop so as to prevent losing ships & crew. That solves:

- [x] No more immense resource drain on a faction that has no visible way to support it.
- [x] No more insane murdering of each other (neither side wants to destroy each other, after all)
- [x] No more easy access to large numbers of disabled ships (there will probably still be a few, but a lot less)
- [x] Shield Beetles would require a bit more time & effort to acquire

**Comment by Vedvod** [comment]
I agree on both counts
Certainly the Unfettereds are aggressive and eager for combat, but they are not reckless, and they are certainly not stupid enough to attack when they know they'll lose. There is no reason for literally entire fleets of Unfettereds to suicide rush when it achieves nothing for them.
On the second, having combat not pause while boarding would be an interesting concept, but as a solution for shield beetle farming, I personally do not think it would work out, since it would just make it harder for a normal player to board a ship normally and not through an attempt to "exploit" this system.
Once again, I push for an Unfettered (or cheap) version of the Shield Beetle with worse stats somehow that does not detract from combat or capture difficulty, but still is worse than the regular Hai one in some way, maybe with cheaper cooling, or less outfit space? 
The original suggestion of decreasing fleet frequency is certainly the best solution in my opinion, since it solves the "The Unfettereds aren't suicidal idiots" lore problem, and it also solves the exploit.

**Comment by movingpictures** [comment]
> There's already a lot of comparisons with the SB and the Albatross being very similar. We don't want to start giving the SB things built into the hull too. That road leads to them becoming even more similar than they already are, and reduces the Albatrosses main advantages.

Very valid points. The general statements from this thread suggest the SB is too powerful.... so I hear the sentiment about not giving it shield regen. 

So my counter would be - OK, if we make the shield beetle available later, then perhaps the mission to claim it is not related to shield regen. (BTW, I kinda like the albatross as a flagship.)

Alternately, What if the Shield Beetle is the vessel developed by the Unfettered as part of an invasion into Wanderer space? (Kinda makes sense a warlike faction would develop a nasty warlike ship, right?) If it enters the game then, it wouldn't be a prolific vessel

**Comment by Zitchas** [comment]
> Alternately, What if the Shield Beetle is the vessel developed by the Unfettered as part of an invasion into Wanderer space? (Kinda makes sense a warlike faction would develop a nasty warlike ship, right?) If it enters the game then, it wouldn't be a prolific vessel

This is *exactly* what the solifuge is. One heavy warship being developed like this is good; two probably not.

Description of the Solifuge:
> Impressed by the Alpha's "Giftbringer", the Unfettered endeavored to build their own fighter carrier to take more advantage of their supply of jump drives.

**Comment by movingpictures** [comment]
> This is _exactly_ what the solifuge is. One heavy warship being developed like this is good; two probably not.

Eh, I've captured a Solifuge once. Wasn't impressed with it compared to what was available at the time.

**Comment by vos26** [comment]
_(Personal experience)_ As of 0.9.13, it is fairly challenging already to switch from human heavy warships to Shield Beetles: transports are way too fragile, and Leviathan/Falcon aren't exactly suitable for boarding SBs. To capture a single SB, I had to complete Deep quests and capture Navy Cruiser first, and bunk-maxed Cruiser was still almost unusable for anything.

So, I fail to see a **balance** issue here. But there is obvious **lore-wise** problem, which doesn't fit neither established Hai lore, nor upcoming Hai reveal campaign. It is not possible to keep up such bloody battle for years! After two days of meatgrinder warfare like this both factions would just run out of capable fleets, and total warfare will begin. Which implies Dresden-like extermination of weaker (Unfettered) faction, because it will become only way to stop the bloodshed.
It is also must be noted that current Unfettered fleets do not reflect their supposed "resource exhaustion", and they fight not to plunder, but to kill. That doesn't make much sense. Why do they throw such large rocket barrages? Even if they has ability to produce them, they should be avoiding rocket usage, because, well, that's what any sane player does when capturing/looting, and Unfettered are supposed to be military geniuses. Normal Hai, on the other hand, have actual working economy behind their backs (isn't Tracker supposed to be technological marvel, at least by human standarts?), and zero motivation to die - so they would be more inclined to use rockets instead of charging with guns blazing.
Adding "last ditch"/"cannon fodder" SB variants to Unfettered doesn't look like clever idea to me. SB is a heavy warship. Suppose you cut costs on shield regen, that will give you up to 1.5M per ship. You still get to pay over 7M for Boulder + Geode reactors, and over 3M for engines, and do not forget hull price! And your warship will be much more likely to die, Hai shield regen is not something to ignore. And it doesn't even fit into set of beliefs of Unfettered: they view themselves as scions of great empire, keepers of true tradition, so they must take great pride in their most advanced ancient weapon, Shield Beetle, they won't go cheap in this issue. Of course, "cannon fodder" tactics would fit into this mindset nicely, such government would gladly throw tons of poorly armed disposable fighters into battle, and that's exactly what Solifuge does. Note that Solifuge is beefier than SB, and it is great lore-wise: drown enemy in corpses of simple soldiers, while commanders stay inside big and impressive capital ship - that's the strategy, which I fully expect to see from schizo-militaristic regime such as Unfettered.

I suggest the following:
1. Add "coward"-like behavior to all normal Hai fleets and to all Unfettered fleets which include SB (they are too costly to lose), as Zitchas suggests. (Optional: add to Unfettered some non-SB fleets without coward behavior, as they must cover SB retreat - their lives cost less than a single SB).
2. Make normal Hai fleets in Cloudfire all rocket-maxed, and make Unfettered fleets spawn rocket ships extremely rare (maybe as part of non-coward "suicide squads", since their orders are to cover SBs retreat). Note that such changes would probably increase difficulty of capturing Unfettered SBs, as Terin suggested: SBs would either flee before disabling, or rockets would end them before player will board them. (Or Unfetterd would overwhelm Fettered defenses for a moment, and player will get huge, angry and well-shielded problem - instead of easy loot).
3. No changes should be made to spawn frequencies. Outfit/behavior changes should be enough to make this conflict sensible, while keeping Cloudfire very dangerous zone for early-game player.

**Comment by movingpictures** [comment]
It's possible to gain a Shield Beetle in about two game-months or so. Cap
pirates until about 7.5 M wealth, buy a Star Queen, go to Hai Space. Cap a
disabled shield beetle. It's ridiculously easy.

**Comment by VonZorch** [comment]
Fairly simple solution and well in keeping with lore.  The Unfettered Hai being dedicated warriors, give them "crew attack" 3 and "crew defense" 6, maybe more, and of course fill all bunks with crew.

**Comment by Hurleveur** [comment]
https://github.com/endless-sky/endless-sky/pull/6906 would this personality be satisfactory as a fix to this issue?
I intend to make the unfettered cowards and the hai merciful once the personality is merged, and trying that out it means they no longer actually kill each others

**Comment by Zitchas** [comment]
I agree. That personality would go a long way toward solving both the lore and balance issue of the Hai/Unfettered warzone.

**Comment by Terin** [comment]
Sounds like it could help, but it might just be a similar issue to the `disabling` personality, where by the time the personality triggers, there's enough hostile ships with enough firepower in the air that they're disabled or dead anyways. I also don't think it'll address the fictional issue of the Unfettered sending hundreds of millions of credits worth of warships and ammunition into a battle they can't win. But I'll try it out, maybe that'll be enough to prevent them from getting into protracted fights.

**Comment by Hurleveur** [comment]
Well the ships try to leave when the shields are down, and dont get shot at so even if they get disabled they wont be destroyed, hopefully

The UHai wasting ressources is normal though

**Comment by binaryspica** [comment]
> Sounds like it could help, but it might just be a similar issue to the `disabling` personality, where by the time the personality triggers, there's enough hostile ships with enough firepower in the air that they're disabled or dead anyways. I also don't think it'll address the fictional issue of the Unfettered sending hundreds of millions of credits worth of warships and ammunition into a battle they can't win. But I'll try it out, maybe that'll be enough to prevent them from getting into protracted fights.

If it were Human, then I agree that they can have some reason and care for manpower and resource. But they are **Unfettered Hai**; they are the race that gladly sending hundreds of millions of credits worth of warships and ammunition into a battle they can't win and it's all natural for them and if they are not then something is gone horribly wrong. Given the various descriptions in the game, I can't think otherwise. and most of them are nothing more than crazy and bloodthirsty lemming. Although there are someone can ask if they were wrong.

**Comment by UnorderedSigh** [comment]
I tested #7471 extensively, and it seems to resolve this issue. 

1. Disabled Unfettered Hai Shield Beetles are extremely rare (every 20-30 minutes) which gives the appearance that the Unfettered aren't losing many ships.
2. The system is still quite lethal to the Fettered and human merchants, which makes the Unfettered's tactics appear successful. 
3. Disabled Lightning Bugs are slightly more common than before, which makes it look like Unfettered are sacrificing cheaper Lightning Bugs to help their expensive Shield Beetles escape.
4. This also eliminates a major problem in game balance: it was too easy to get a fleet of capital ships, expensive outfits, and millions of credits per minute. With the new changes, you need serious combat capabilities to do it.
5. You now have to paralyze a Shield Beetle long enough to disable it, or it will run away. That's hard to do without the right weapons or tactics. This makes it look like the Unfettered care *a lot* about the survival of their ships.

This all happened because @Hurleveur updated the code so Unfettered are `coward`s and the Fettered are `merciful`. That makes a damaged Unfettered ship run away, and the Fettered let them.

**Comment by binaryspica** [comment]
> I tested #7471 extensively, and it seems to resolve this issue. The Unfettered tactics now appear clever, cost-effective, and successful, despite the extreme chaos at the system.
> 
>     1. Disabled Unfettered Hai Shield Beetles are extremely rare (every 20-30 minutes) which gives the appearance that the Unfettered aren't losing many ships.
> 
>     2. The system is still quite lethal to the Fettered and human merchants, which makes the Unfettered's tactics appear successful.
> 
>     3. Disabled Lightning Bugs are slightly more common than before, which makes it look like Unfettered are sacrificing cheaper Lightning Bugs to help their expensive Shield Beetles escape.
> 
>     4. This also eliminates a major problem in game balance: it was too easy to get a fleet of capital ships, expensive outfits, and millions of credits per minute. With the new changes, you need serious combat capabilities to do it.
> 
>     5. You now have to paralyze a Shield Beetle long enough to disable it, or it will run away. That's hard to do without the right weapons or tactics. This makes it look like the Unfettered care _a lot_ about the survival of their ships.
> 
> 
> This all happened because @Hurleveur updated the code so Unfettered are `coward`s and the Fettered are `merciful`. That makes a damaged Unfettered ship run away, and the Fettered let them.

Is the disabled means disabled without your own effort and only by Hai fleets? If so then it is not that bad. As you mentioned that it only changed the behavior, not the amount of generated Shield Beetles so as long as it they keep it it would be fine. Then you may still hunt for Shield Beetle by your own fleet on Wah Ki.

**Comment by MasterOfGrey** [comment]
> The only issue is it is not actually meet the lore, for Unfettered are not the sane people with a reason. They are far from a coward, more like remmings and sucidal maniac.

The lore for the unfettered was paper thin. We have deepened it and this is no longer a correct and accurate interpretation.

**Comment by UnorderedSigh** [comment]
> Is the disabled means disabled without your own effort and only by Hai fleets?

The NPCs rarely disable Unfettered Shield Beetles without your help. You can disable them yourself, but it is harder now that they try to run away.

This is how they're disabled without your help:

1. Thousands of damage of trackers still in flight after Fettered ships stop shooting. Usually the beetle will leave or die, but a rare few will be disabled.
2. Unwise human merchants, who usually die before they can do significant damage.
3. Being on top of exploding ships, but the beetle has to already be severely damaged, and not damaged enough to die from it (a narrow range).

To disable them yourself, there's an added challenge: you have to paralyze them or push them to stop them from leaving, while also damaging them. Humans and Korath have excellent options for this.

**Comment by Zitchas** [comment]
Honestly, I never saw the unfettered as berserk suicidal combat-addicts, but rather entirely sane people who happen to be significantly more aggressive than the regular Hai population. In political speech, they'd be termed "ultra-nationalist hawks." Even "Imperialist nationalists."

Regarldess of what the *code* for making people run away after taking a lot of damage is called, this isn't actually cowardly behavior, but rather tactically intelligent behavior. They're trying to impress their brethren with military prowess (and maybe get them addicted to the adrenalin/thrill of battle), not kill as many of their own people as possible. They want to convince more people to join them (and capture supplies), not weaken their own country.

So as far as I'm concerned, the change in 7471 is simply making the in-game viewed action match the lore, rather than radically changing the lore.

**Comment by UnorderedSigh** [comment]
The `coward` AI is a misnomer. It isn't cowardly; it causes the ship to retreat when it is damaged, as any ship would in the real world. Without it, the ship will keep fighting even if it is 1 hull point away from being disabled. Really, it is the "normal person AI" while a non-`coward` ship has the "crazy psycho killer" or "heroic death" AI.

**Comment by Hurleveur** [comment]
Actually, no, normal ships retreat at 25% health, cowards at 50%. All raiders need to be coward or they lose more than they gain by raiding.

The Unfettered that were berserk were those we saw throw themselves to explode in wah ki for many years.

Seriously though some Unfettered were probably berserk but after a few decades of fighting they're all dead.

As for farming SB you need to be well equipped now and cant expect the Hai to do it all for you anymore.

**Comment by mathwhiz1212** [comment]
A new player's perspective: Capturing my first shield beetle at Wah Ki was very costly (lost 9 aprosys, I learned they're very susceptible to ion blasts) and took three tries even with four heavy warships and 10-15 aprosys. Second beetle was easier, we'll see from there.

**Comment by UnorderedSigh** [comment]
> Capturing my first shield beetle at Wah Ki was costly

Yes, the intent of the change is for the first one to be difficult. Once you have one shield beetle, you can outfit it to disable and capture another without needing any escorts. If you have a few beetles with weapons that push enemies (ion cannon, particle cannon, proton weapons, banisher, expeller) you can capture more very quickly.

I actually found Lightning Bugs with proton weapons were great at capturing the first Beetle. You can get them in bulk easily with a solo medium warship that has proton or particle weapons. It's a lot of fun too, and a good challenge, dodging the beetles and trapping a lightning bug. 

Lightning Bugs sell for 2-ish million credits each, so you can save up cash quickly, if you can fight for it. Not as easy as farming Boulder Reactors used to be, but almost as fast with the right ship.

**Comment by Hurleveur** [comment]
Well you played it fair, actually fighting instead of relying solely on hai, which you should still be able to do

**Comment by binaryspica** [comment]
> A new player's perspective: Capturing my first shield beetle at Wah Ki was costly (lost 9 aprosys, I learned they're very susceptible to ion blasts) and took three tries even with four heavy warships and 10-15 aprosys. The second beetle was captured at a calm moment. The next two cost me a Leviathan fully equipped with the best of Hai & human tech. Still worth it, but costly.

Since it is normal that lost a Human or Hai ship on there, that's not so bad. Though that's not actually about the main topic; capturing Shield Beetle with only a marine flagship and it alone....

**Comment by MasterOfGrey** [comment]
> Since it is normal that lost a Human or Hai ship on there, that's not so bad. Though that's not actually about the main topic; capturing Shield Beetle with only a marine flagship and it alone....

There's going to need to be a better rework of boarding mechanics (one is proposed and awaiting dev consideration) to *really* hash down the issue to edge case luck, but I think *most* of the other possible things which can be done have now been done.

**Comment by Hurleveur** [comment]
Oh not really I have more in store
Aka missions that bring reinforcements or blow the ship up when you board it

**Comment by mathwhiz1212** [comment]
@binaryspica

> only a marine flagship and it alone

It was part of a fleet of a shield beetle and three heavy warships from human space. But I can still roll in with six shield beetles and three heavy warships and get totally destroyed. So I tend to agree with the point of Wah ki being too much of a warzone (at least for someone not super familiar with advanced combat tactics).

**Comment by UnorderedSigh** [comment]
> six shield beetles and three heavy warships and get totally destroyed

The Unfettered raiding fleets are designed to disable and pillage capital ships, especially Shield Beetles. If they weren't, then the Unfettered wouldn't be raiding Wah Ki in the first place. By fighting in Wah Ki without advanced combat tactics, you're volunteering to be a target of an Unfettered raid.

You're describing Wah Ki behaving as expected: Unfettered raiding parties are able to effectively disable or destroy strong targets, without taking significant casualties.

**Comment by mathwhiz1212** [comment]
@UnorderedSigh An alternate solution for me would be telling me what advanced tactics I need to learn. It's not simply that I'm annoyed at getting decimated, I'm annoyed at not knowing how to learn how to not be decimated.

> > six shield beetles and three heavy warships and get totally destroyed
> 
> You're describing Wah Ki behaving as expected: Unfettered raiding parties are able to effectively disable or destroy strong targets, without taking significant casualties. The Unfettered raiding fleets are designed to disable and pillage capital ships, especially Shield Beetles. If they weren't, then the Unfettered wouldn't be raiding Wah Ki in the first place. By fighting in Wah Ki without advanced combat tactics, you're volunteering to be a target of an Unfettered raid.

**Comment by UnorderedSigh** [comment]
Stay out of the action until the right moment. That requires strong, fast, ships. Don't let your escorts wander off; even the strongest ships won't survive long if they get cornered. The best way to do this is to have them gather around your flagship until the right second.

If you want to disable a Shield Beetle, you need to push it out of the action and damage it without destroying it. Particle Cannons are the best weapons for this. Proton weapons are good too, and Ion Cannons will work in a pinch (and they come standard on many Shield Beetles). For something more exotic, you can use Korath Expellers or Banishers along with something of similar range to damage it (like a Korath Inferno or Sunbeam).

It works best if all of your ships are much faster than Shield Beetles. My favorite approach is a Shield Beetle with the default steering, and Korath Planetary-Class Thrusters (easy to get in Remnant space or pirate systems beyond Alcyone). You can supplement it with a little Coalition steering once you can reach them.

For capturing a Shield Beetle, a Star Queen, Centipede, or Shield Beetle will work, if you add extra bunks and some good engines. Don't use a Bactrian or Kimek Spire; they're too slow and they're big targets. Make sure you have enough pulse rifles (or better rifles) for all crew. Add Fusion Grenades if you have the cash. Never bring Nerve Gas into Hai space; it isn't worth the fines.

The same tactics work with Lightning Bugs, which you can capture very quickly with a medium warship.

**Comment by mathwhiz1212** [comment]
Thank you! That was very helpful. Actually capturing the ship after it was disabled and out of the fight wasn't a problem, it was getting to that point. You made some good suggestions. Maybe someone should put that in a FAQ or guide somewhere.

This is an actionable non-cowardly strategy. For my last beetle captures I got tired of fighting so I just hid until the Hai disabled one and swooped in to take it. I'd prefer to do it by skill.

> Stay out of the action until the right moment. That requires strong, fast, ships. Don't let your escorts wander off; even the strongest ships won't survive long if they get cornered. The best way to do this is to have them gather around your flagship until the right second.
> 
> If you want to disable a Shield Beetle, you need to push it out of the action and damage it without destroying it. Particle Cannons are the best weapons for this. Proton weapons are good too, and Ion Cannons will work in a pinch (and they come standard on many Shield Beetles). For something more exotic, you can use Korath Expellers or Banishers along with something of similar range to damage it (like a Korath Inferno or Sunbeam).
> 
> It works best if all of your ships are much faster than Shield Beetles. My favorite approach is a Shield Beetle with the default steering, and Korath Planetary-Class Thrusters (easy to get in Remnant space or pirate systems beyond Alcyone). You can supplement it with a little Coalition steering once you can reach them.
> 
> For capturing a Shield Beetle, a Star Queen, Centipede, or Shield Beetle will work, if you add extra bunks and some good engines. Don't use a Bactrian or Kimek Spire; they're too slow and they're big targets. Make sure you have enough pulse rifles (or better rifles) for all crew. Add Fusion Grenades if you have the cash. Never bring Nerve Gas into Hai space; it isn't worth the fines.
> 
> The same tactics work with Lightning Bugs, which you can capture very quickly with a medium warship.

**Comment by Hurleveur** [comment]
I could tweak the spawn rate from 550 to 600 but its meant to be chaos not a shield beetle farm

**Comment by UnorderedSigh** [comment]
I discourage any further changes in this PR. As it is now, the Unfettered have the appearance of raiding effectively, and if they player gets in the way, it'll still seem like an effective raiding force. It is difficult to farm, even for a beetle farming pro like me. You really need a squadron of specialized ships to do it quickly ([Wacky Wahki~3014-09-06 snatcher and four pushers.txt](https://github.com/endless-sky/endless-sky/files/10013758/Wacky.Wahki.3014-09-06.snatcher.and.four.pushers.txt)). Otherwise, it is a hard struggle and you'll have to load frequently due to losing sizable chunks of your fleet ([Wacky Wahki~3014-05-30 bug queen.txt](https://github.com/endless-sky/endless-sky/files/10013765/Wacky.Wahki.3014-05-30.bug.queen.txt)). 

An occasional massive raid fleet with specialized weapons is a great idea. Please do that, but not in this PR. It does already feel like there are massive raid fleets occasionally though, since the random number generator will spawn several fleets at once.

**Comment by UnorderedSigh** [comment]
I'll post some more fleets in 5-10 hours for people who want to do farming tests and survival tests in the system.

**Comment by UnorderedSigh** [comment]
These solo ships and fleets are good at farming in Wah Ki.

**Solo Beetle Snatcher**
Once you have one Shield Beetle, it's easy to get more. This Beetle can mass farm more Shield Beetles for you. There are many possible variations on this design. The critical parts are: many bunks, enough pulse rifles & grenades for everyone, and six particle cannons. A Tactical Scanner can help you find Beetles with a low crew count. This Beetle can probably handle about 80 crew on the target ship before risking being undercrewed after a capture. Just fly around until there's a vulnerable lonely beetle, disable it, capture it, and jump away before it's destroyed.
[Wacky Wahki~3014-12-27 solo beetle.txt](https://github.com/endless-sky/endless-sky/files/10017459/Wacky.Wahki.3014-12-27.solo.beetle.txt)

**Sparrow Bug-Eater**
This Sparrow design can farm Lightning Bugs at a rate of about 1 every 5 minutes in fast-forward. Note the lack of a Hyperdrive. That lets you reach 15 bunks, but you'll need to bribe the Unfettered every time before landing. If you want to keep all the Lightning Bugs, you can sell unwanted weapons from them to pay for the continued bribes.
[Wacky Wahki~3014-01-01 have one lightning bug.txt](https://github.com/endless-sky/endless-sky/files/10017471/Wacky.Wahki.3014-01-01.have.one.lightning.bug.txt)

**Early-Game Firebird**
This one is for farming lightning bugs. About half of them you'll find for free, as the Sparrow did, but the rest you can disable. This is a low-tech but sufficient Firebird design that can farm Lightning Bugs through combat. Expect to die half the time you go out, but it's still a fast capture rate. (And highly entertaining.) There are probably better builds, but this is one you can get in the early game through capture and pillage. (I never buy ships.)
[Wacky Wahki~3014-03-13 six bugs.txt](https://github.com/endless-sky/endless-sky/files/10017498/Wacky.Wahki.3014-03-13.six.bugs.txt)

**Bug Queen**
A somewhat quick way to get your first Shield Beetle is with six Lightning Bugs that have proton weapons, plus a Star Queen. That's because you can get the Bugs quickly, and a Keystones visit to the Remnant will give you most of the money you need for the Star Queen. It'll take several tries since you need to corner an injured beetle and paralyze it long enough to disable it, before someone wipes out your Lightning Bug fleet. After you have that first beetle though, you can capture more easily.
[Wacky Wahki~3014-05-30 bug queen.txt](https://github.com/endless-sky/endless-sky/files/10017508/Wacky.Wahki.3014-05-30.bug.queen.txt)

**Snatcher and Pushers Mk. 1**
The flagship is a durable troop transport (a Shield Beetle) and the four escorts are durable Shield Beetles with 8 particle beams each. You can push other beetles out of the action pretty quickly if you come in at the right angle. Keep the fleet together though, or you'll lose ships. 
[Wacky Wahki~3014-09-06 snatcher and four pushers.txt](https://github.com/endless-sky/endless-sky/files/10017519/Wacky.Wahki.3014-09-06.snatcher.and.four.pushers.txt)

**Comment by UnorderedSigh** [comment]
Next round of farmers and survival tests:

**Faster Mass Beetle Farming: Snatcher and Pushers Mk. 3**
This is a faster large-scale farming fleet; about 70 beetles per hour. One Beetle is equipped for capture, and the other four have a mix of Expellers and proton weapons.
[Friction Diction~3025-11-29 proton-expeller snatcher.txt](https://github.com/endless-sky/endless-sky/files/10018480/Friction.Diction.3025-11-29.proton-expeller.snatcher.txt)

**Rano'erek + KIV 349s**
Try the Unfettered against a quartet of heavily upgraded Tier 2 capital ships. All five ships have afterburners, and your four escorts have a powerful combination of outfits designed to disable anything in seconds. Yet, somehow the Unfettered eventually destroy them. 
[Friction Diction~3025-12-01 rano-349.txt](https://github.com/endless-sky/endless-sky/files/10018508/Friction.Diction.3025-12-01.rano-349.txt)
Try this:
1. Let your KIV 349s run around killing Shield Beetles. Eventually, one of the 349s will be swarmed by a large Unfettered raid and die.
2. Guide your KIV 349s with you, ensuring they strike the same target and stay together. They'll last much longer, and you'll have to wait until a truly gigantic fleet arrives before you lose any.
3. Use your afterburners any time a gigantic fleet comes around. The Shield Beetles will find another target while your escorts repair.

**Armada**
Here you can stress-test the capabilities of the Unfettered. There's 124 capital ship escorts with tech from all over the galaxy. Sadly, the Unfettered are not up to the task.
[Friction Diction~3025-12-04 everything.txt](https://github.com/endless-sky/endless-sky/files/10018527/Friction.Diction.3025-12-04.everything.txt)

**Comment by Hurleveur** [comment]
Wow. Thx for providing such detailed testing.
I think all of that is great balance wise

**Comment by UnorderedSigh** [comment]
I think you slowed down speed-running. I'm not a speed-runner, but it takes me about 20 minutes longer to get my first 5 Beetles.

The fastest method I've found is:

1. Sparrow captures another Sparrow to replace original, plus capture and sell other things (can do all of this in the same day)
2. Sparrow captures 2 Lightning Bugs
3. Sell Sparrow and one bug, capture a Firebird. Sell the bug and use its parts and some money to upgrade the Firebird
4. "Assist" the defenders at Cinxia with emotional support while pillaging outfits from Korath they disable, including at least 2 jump drives.
5. Capture a big automata ship, and give it a mix of outfits that push and damage. Upgrade your Firebird along the way, so you stop dying. Also, get 14 mil credits (you'll probably have that already from "assisting" the defenders).
6. Buy a Centipede, and outfit it for capture.
7. Capture five beetles, iteratively adding them to the fleet each time (4x Ion Cannon, 2xpulse turret, 2xanti-missile)

It's a bit game-exploity, but the same is true of using an unarmed troop transport to mass farm Shield Beetles.

**Comment by Hurleveur** [comment]
Well slowing it down was the goal,and I'm not done with it yet.
There will be boarding missions and better rifles for the Unfettered probably as well

**Comment by UnorderedSigh** [comment]
> slowing it down

Actually, mass farming is slightly faster now that the Fettered stop killing the ships, and the Unfettered stop fighting at 50% health. What is harder is setting up the mass farming fleet.

**Comment by UnorderedSigh** [comment]
If you want to slow down mass farming, you'll need better AI. The ships need to fight back when they can't jump away, or ships will have to come to each other's aid.

**Comment by Hurleveur** [comment]
When boarding the ships unbribable reinforcements (ignored by the Hai) will arrive or the ship will blow up, if the mission triggers. Sometimes nothing would happen ofc, it'll depend on your crew count relative to theirs

**Comment by Hurleveur** [comment]
https://github.com/endless-sky/endless-sky/pull/7620 here we go that should fix this issue completely, with the other PR and the incoming proper raiding fleets once the other PR is merged.
You'll still be able to loot ships +- but not to capture them as easily. (having no more crew than the enemy guarantees nothing happens)

**Comment by Terin** [comment]
Idled around in Wah Ki for a good while, and it is MUCH better in terms of ships getting disabled. I think there's still a fair amount of murder going on, but it's much less obvious when tons and tons of disabled ships are kept around. Being able to have like, NPCs hold off on missile barrages if enough munitions are in the air to disable/kill a target. But besides that, this is a massive improvement to what we had enough, where I'm willing to have this issue closed.

**Comment by mathwhiz1212** [comment]
It seems more chaotic in v0.10.0 than v0.9.16. In a half hour to an hour on fast-forward I captured a Sea Scorpion only.

Removing the `coward` flag seems to make it more possible. Ships are destroyed very quickly, but I can capture a Sea Scorpion or Lightning Bug every few minutes on fast forward. It appears I may need a larger force to capture a beetle. 

With the `coward` flag, it would be much more effective to capture ships pretty much anywhere else.

**Comment by EjoThims** [comment]
Capturing fewer ships here is indeed desired and intended behavior.

**Comment by mathwhiz1212** [comment]
> Capturing fewer ships here is indeed desired and intended behavior.

Correct.

My impression was that a few minutes per capture was the goal, but in my experience with the `coward` flag it's simply not worth capturing **any** ships in Wah Ki. If I'm going to spend an hour, I want to cap a tier 2 ship, not a Hai medium warship.

That's what I was wondering about. Was the goal to limit capping in Wah Ki or essentially eliminate it entirely?

**Comment by EjoThims** [comment]
Goal was to reduce as low as possible the ability to passively capture ships (shield beetles especially) from this system.

I've not myself seen it reduced so much as you suggest (I myself indeed see the "few minutes per capture" if I'm willing to just grab anything available), but I consider 0 passive ship captures to be explicitly a win for the goal of "fewer passive ship captures."

**Comment by Hurleveur** [comment]
The fact less get disabled means it is less "too much of a warzone"? What that partly meant is too many ships died.

Funny thing: I joined this project with fixing this huge balance issue in mind. Took a while and I got some other things to do along the way.

You can easily pursue a ship and capture it. The Hai should not help you do it. Now they wont detroy them and UHai run away as they should, even if that means they are easier to defeat in battle.
Just get an actual warship and do smth if you want a ship. They are no longer free!

**Comment by Zitchas** [comment]
It's long been considered an unbalanced, and highly inappropriate for the lore, that the player can easily snag the best Hai Heavy Warship for free with zero effort; and not only one, but entire fleets of them without doing much more than having the patience to wait a few minutes. 

a) Unfettered have limited resources and lives. They shouldn't be fighting to the death to prove a point.
b) The Wah Ki battle is basically the Unfettered pushing the regular Hai to maintain a warfleet. Essentially a "If you aren't going to properly maintain defenses against the aliens we know are out there, then *we* will be the active threat to force you to do so." They don't want to kill the Hai, nor destroy their ships. They just want to provide enough of a threat to force the Hai to develop and maintain a reasonably large military. And they need supplies, so anything they can raid here is useful, too.
c) The Hai don't hate the Unfettered, nor do they seek their destruction. In fact, they even provide them with food shipments, and harbor some sympathies for them. Likewise, the Unfettered don't hate the Hai.

These three points of lore combine to mean that the Unfettered should be running away when they are seriously damaged, the Hai should let them (not pursuing them to destruction), and both sides avoiding "fight to the death" engagements, nor seeking to utterly destroy each other.

**Comment by binaryspica** [comment]
Honestly, it won't affects much for the most players, for it only changes the behaviors but the spawn rate is left untouched. I bet that almost all players who visit Wah Ki first times are expected to get a plenty of warship escorts enough to knockout a shield beetle already. Remember that found the wormhole is not an easy task if you have no information about that at all. If you don't aware that there is the race called Hai, you only found them if you already have visited every single hunam territory, got tired then want to check every single boarder of the human space.

**Comment by Zitchas** [comment]
Well, some of us like to explore. I found the Hai long before I got into a Heavy Warship, or even considered having a fleet. Wah Ki was just a deathzone for me flying what was effectively a a high-speed scoutcraft.

**Comment by Zitchas** [comment]
The biggest nerf is probably versus the veterans or people following guides expecting to be able to pick up a heavy warship on the cheap with a Star Queen. $6M investment (and a little luck avoiding pirates on the way there) to get a fleet of $33M ships? can't beat that sort of opportunity.

**Comment by Hurleveur** [comment]
The idea wasnt to delete this gameplay element, just to make it more normal.
The fleets did get an increase tho

If needed it can be deleted with the flick of a button by making the Hai hate you for capping UHai

**Comment by UnorderedSigh** [comment]
It's actually about as easy to mass-farm Shield Beetles as it was before, if you use particle cannons. What has changed is that it takes about 15-30 minutes longer to get your first Beetle since you need to build up to it first, rather than just doing the Keystone mission and buying a Star Queen.

**Comment by Hurleveur** [comment]
Ok, they will def mind it somewhat, mb even with a small mission triggering to warn you.

**Comment by binaryspica** [comment]
> The biggest nerf is probably versus the veterans or people following guides expecting to be able to pick up a heavy warship on the cheap with a Star Queen. $6M investment (and a little luck avoiding pirates on the way there) to get a fleet of $33M ships? can't beat that sort of opportunity.
> 
> If people already have a dreadnought and/or a warfleet, then they've already invested enough that getting a shield beetle is both accessible and probably even just straight up affordable for them - or could be fairly easily. The main problem that this was trying to solve was the former case of being able to "luck into" a best-in-game fleet with nothing but a good boarding ship. (best not because it's the best ship, but best because it's the easiest to achieve in huge numbers. Or was, anyway)

Those 'veterans' are expected to get a plenty of ships nonetheless, or they start a game with full of knowledge about the game so they could do better. I don't think that it have any impact for them either, since it is not so difficult to have one to five human heavy warships with the flagship to enter Wah Ki if you are start a game but already played the game a lot.

**Comment by UnorderedSigh** [comment]
> Those 'veterans' are expected to get a plenty of ships nonetheless, or they start a game with full of knowledge about the game so they could do better. I don't think that it have any impact for them either, since it is not so difficult to have one to five human heavy warships with the flagship to enter Wah Ki if you are start a game but already played the game a lot.

This is true, but for habitual farmers like me, the new Wah Ki adds a fascinating new challenge: how little do I need to get a Shield Beetle? Before these changes, this was a solved problem. Now, not only are there completely new challenges, but there is a new Unfettered ship to farm. I expect that making farming more exciting wasn't the goal of the Wah Ki changes, but I appreciate it nonetheless.

**Comment by mathwhiz1212** [comment]
I agree it was too easy to cap shield beetles passively before, but I find it interesting that for me at least it's easier to cap a Korath Rano'erek than a shield beetle early -ish game.

**Comment by UnorderedSigh** [comment]
> I agree it was too easy to cap shield beetles passively before, but I find it interesting that for me at least it's easier to cap a Korath Rano'erek than a shield beetle early -ish game.

There are individuals working to make Korath farming more eventful as well.

**Comment by mathwhiz1212** [comment]
> > I agree it was too easy to cap shield beetles passively before, but I find it interesting that for me at least it's easier to cap a Korath Rano'erek than a shield beetle early -ish game.
> 
> There are individuals working to make Korath farming more eventful as well.

Korath farming feels like a winnable challenge at this point. I've never really mastered the strategy of active shield beetle capping.
